public/app/plugins/panel/table/table-new/TableCellOptionEditor.tsx

```diff
@@ -25,6 +25,19 @@ interface Props {
   onChange: (v: TableCellOptions) => void;
 }
 
+const TEXT_WRAP_CELL_TYPES = new Set([
+  TableCellDisplayMode.Auto,
+  TableCellDisplayMode.Sparkline,
+  TableCellDisplayMode.ColorText,
+  TableCellDisplayMode.ColorBackground,
+  TableCellDisplayMode.DataLinks,
+  TableCellDisplayMode.Pill,
+]);
+
+function isTextWrapCellType(value: TableCellOptions): value is TableCellOptions & TableWrapTextOptions {
+  return TEXT_WRAP_CELL_TYPES.has(value.type);
+}
```

Comment:
I don't like having a hardcoded list of which cell types get the option which duplicates the cue mapping, but I don't know of a good alternative since `wrapText` is an optional boolean.